src/shared/lib/pathwayMapper/PathwayMapperTable.tsx

```diff
@@ -126,13 +132,13 @@ export default class PathwayMapperTable extends React.Component<
         };
 
         this._columns[IPathwayMapperTableColumnType.SCORE] = {
```

Comment:
Use the `columnsOverride` props here, assuming that it is like

```
{
    [IPathwayMapperTableColumnType.SCORE]: {
        name: '# Genes matched',
        tooltip: <span>Number of Genes Matched</span>
    },
    ...
}
```

```suggestion
        this._columns[IPathwayMapperTableColumnType.SCORE] = {
            ..., // current default values (no need to modify)
            ...this.props.columnsOverride[IPathwayMapperTableColumnType.SCORE] // override values
        }
```